src/features/content-explorer/content-explorer/constants.js

```diff
@@ -0,0 +1,23 @@
+import { TYPE_FOLDER } from '../../../constants';
+
+/* ----------------------- Helpers for ContentExplorer ---------------------------- */
+
+export const areAllItemsActionDisabled = items => {
+    const result = items.every(item => {
+        return item.isActionDisabled;
+    });
+    return result;
```

Comment:
this could return directly
```suggestion
    return items.every(item => item.isActionDisabled);
```